### PR TITLE
Feat/xd-153 -  Label Components properly in Create Case Page with CSS

### PR DESCRIPTION
**Pull request by Sabo2k**
Refactored Create Case Page and improved labels of the form

### CODE REVIEW

**Comment by Sungonogi** [comment]
The #153 issue is not "add filter options for the cases page" but something else. Did you pick the wrong name?

**Comment by Sabo2k** [comment]
> The #153 issue is not "add filter options for the cases page" but something else. Did you pick the wrong name?

Yes, you're right. My bad. Thanks for letting me know 👍

File: libs/cases/frontend/view/src/lib/components/create-case/create-case.component.html [hunk 1]
```diff
@@ -7,51 +7,74 @@
             #caseForm="ngForm"
             (submit)="onSubmit(caseForm)"
         >
-
+            <!-- Facility Selection -->
             <ix-select
                 id="facilitySelection"
                 (valueChange)="onFacilityChange($event)"
                 (inputChange)="onFacilityInputChange($event)"
+                hide-list-header="true"
```
Comment (Sungonogi) [inline libs/cases/frontend/view/src/lib/components/create-case/create-case.component.html]: If I see it correctly that hide-list-header removes the "Please select an option" showing at the top of the selection
grafik

If you don't like it we can definitely remove it but you should also do it for the other  2 \<ix-select\> then

**Comment by Sungonogi** [comment]
I have to say in my opinion these HTML comments like \<!-- Date Selection- -\> are not very useful, because most of the time HTML code is self explanatory (and we don't have them anywhere else). But since they might be helpful in some cases it would be ok for me if you keep them.

**Review by Sungonogi — CHANGES_REQUESTED** [review]
Only thing I found was the hide-list-header. Apart from that it looks good.

**Review by Sungonogi — APPROVED** [review]

File: libs/cases/frontend/view/src/lib/components/create-case/create-case.component.html (hunk 1 above)
Comment (Sungonogi) [reply libs/cases/frontend/view/src/lib/components/create-case/create-case.component.html]: Ty for removing it

**Comment by Sungonogi** [comment]
There are so many files changed eventhough you only actually changed one in a meaningful way. Sometimes this happens eventhough there is no difference to develop, but here there is a [difference](https://github.com/amosproj/amos2024ss01-xcelerator-demo-app/compare/develop...feat/xd-153). Since you only changed one file I suggest that you copy that somewhere then completely reset this branch or create a new branch of develop. Then paste that file on the reseted/new branch and merge that

**Pull request closed without merging**